### PR TITLE
Add missing plugindescriptions

### PR DESCRIPTION
**Pull request by rdbende**
Add missing plugindescriptions for black, directory_tree and sort plgins.

### CODE REVIEW

File: porcupine/plugins/directory_tree.py [hunk 1]
```diff
@@ -1,3 +1,9 @@
+"""
+Displays a directory tree on the left side of the editor with the folders containing the 5 most recently edited files.
```
Comment (Akuli) [inline porcupine/plugins/directory_tree.py:2]: A couple problems with 5 most recently:
- The 5 most recent things are projects, not files. For this plugin, "project" and "directory" are slightly different concepts.
- The 5 is not true in all cases. There can be more or less. See the comment above `MAX_PROJECTS`.

I think a good solution is to just delete this:

```suggestion
Displays a directory tree on the left side of the editor.
```

File: porcupine/plugins/sort.py [hunk 2]
```diff
@@ -1,3 +1,9 @@
+"""
+Sorts the selected rows, or if there's no selection, to the next blank line.
```
Comment (Akuli) [inline porcupine/plugins/sort.py:2]: A few nits:
- Sorting happens alphabetically, which is kind of dumb in many contexts, and not necessarily what you want. That should be mentioned.
- rows --> lines. This seems to be more standard and Porcupine already shows "Line 123, column 45" in status bar.
- The logic isn't "to the next blank line". It finds the previous and next blank line, and sorts between them.

```suggestion
Sort the selected lines alphabetically, or if there's no selection, sort between surrounding blank lines.
```

**Comment by rdbende** [comment]
Thanks for the suggestions!. The thing is, I actually wrote these based on my experience, so what others see during use, not what the code exactly does.

**Comment by Akuli** [comment]
Thanks!

**Pull request merged**